packages/api/src/__tests__/e2e/mapi/consolidated.ts

```diff
@@ -1,49 +1,70 @@
-import { Bundle } from "@medplum/fhirtypes";
+import { AllergyIntolerance, Bundle } from "@medplum/fhirtypes";
+import { makeDocumentReference } from "@metriport/core/external/fhir/document/__tests__/document-reference";
+import { metriportDataSourceExtension } from "@metriport/core/external/fhir/shared/extensions/metriport";
+import { PatientWithId } from "@metriport/core/external/fhir/__tests__/patient";
+import { uuidv7 } from "@metriport/core/util/uuid-v7";
 
-export const createConsolidated = (patientId: string): Bundle => ({
-  resourceType: "Bundle",
-  type: "collection",
-  entry: [
+export function createConsolidated(patient: PatientWithId): Bundle {
+  const extension = [metriportDataSourceExtension];
+  const entry: Bundle["entry"] = [
     {
-      resource: {
-        resourceType: "AllergyIntolerance",
-        meta: {
-          profile: ["http://hl7.org/fhir/us/core/StructureDefinition/us-core-allergyintolerance"],
-        },
-        clinicalStatus: {
-          coding: [
-            {
-              system: "http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical",
-              code: "active",
-            },
-          ],
-        },
-        verificationStatus: {
-          coding: [
-            {
-              system: "http://terminology.hl7.org/CodeSystem/allergyintolerance-verification",
-              code: "confirmed",
-            },
-          ],
+      resource: { ...makeAllergyIntollerance({ patient }) },
+    },
+    {
+      resource: { ...makeDocumentReference({ patient, extension }) },
+    },
+  ];
+  return {
+    resourceType: "Bundle",
+    type: "collection",
+    total: entry.length,
+    entry,
+  };
+}
+
+export function makeAllergyIntollerance({
```

Comment:
Should this be added to core as well?

Reply:
So far we've been taking an approach to move to core only what we need, and this is only being used on the MAPI E2E test.

We can chat about how to address core vs. api and other packages (that's a good topic to chat about again on a tech talk), but I think this out of the scope of this PR.